app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/GGMapGestureListener.java

```diff
@@ -11,7 +11,7 @@
 import com.teamagam.gimelgimel.app.view.viewer.gestures.SimpleOnMapGestureListener;
 
 public class GGMapGestureListener extends SimpleOnMapGestureListener
-        implements OnMapGestureListener {
```

Comment:
This  interface declaration is important. I'd like to know when Im looking at it that implements the OnMapGestureListener without going inside SimpleOnMapGestureListener. Moreover, if at any time someone will change/remove SimpleOnMapGesutreListener, Id like the refactoring person to know that this class should still implement the required methods. !fix